src/components/App.js

```diff
@@ -1,16 +1,110 @@
 import React from 'react';
+// import '../styles/App.scss';
+import Menu from './Menu';
+import MakeOrder from './MakeOrder';
+import OrderAdmin from './OrderAdmin';
 
-import '../styles/App.scss';
+import '../styles/base/base.scss';
+import '../styles/components/app.scss';
 
 class App extends React.Component {
   constructor(){
     super();
+
+    this.state = {
+      menu: [],
+      currentOrder: [],
+      orders: []
+    }
+    this.receiveItemOrder = this.receiveItemOrder.bind(this)
+    this.removeItemOrder = this.removeItemOrder.bind(this)
+    this.fetchMenu = this.fetchMenu.bind(this)
+    this.receiveOrderAdmin = this.receiveOrderAdmin.bind(this)
+    this.fetchOrders = this.fetchOrders.bind(this)
+    this.sendOrderToApi = this.sendOrderToApi.bind(this)
+  }
+
+  fetchMenu() {
+    const api = "/api/menu/"
+    fetch(api)
+    .then(response => response.json())
+    .then(content => {
+      this.setState({menu: content});
+    })
+  }
+  fetchOrders() {
+    const api = "/api/orders/"
+    fetch(api)
+      .then(response => response.json())
+      .then(content => {
+        this.setState({orders: content})
+      })
+  }
+
+  sendOrderToApi() {
+    fetch("/api/orders", {
+      method: "POST",
+      body: JSON.stringify(this.state.currentOrder),
+      headers: {
+        "Content-Type": "application/json"
+      }
+    })
+      .then(response => response.json())
+      .then(order => {
+        alert(`We are making stuff for you! Your order ID is ${order.orderid}`)
+      });
   }
 
+  componentDidMount() {
+   this.fetchMenu();
+   this.fetchOrders();
+   }
+
+  receiveItemOrder(order) {
+    const updatedOrder = Object.assign({}, this.state.currentOrder, { [order.id]: order } )
+    this.setState({currentOrder: updatedOrder})
+  } 
+  removeItemOrder(id) {
+    const array = [...this.state.currentOrder];
```

Comment:
It might be easier to store orders in an object that uses ids as keys as it would make look ups much simpler